src/router/agent.router.module.ts

```diff
@@ -23,14 +23,15 @@ export class AgentRouterModule implements NestModule {
      * TODO figure out how to handle case where there is no agent - queue their messages?
      */
     configure(consumer: MiddlewareConsumer) {
-        consumer.apply(createProxyMiddleware('/v1/router',
+        const middleware: any = createProxyMiddleware('/v1/router',
```

Comment:
is this change required because of node + package version changes or just refactor?

Reply:
Ya for some reason on the latest version of nestjs it didn't like the type, but things work fine without it